roadie-rails.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "railties", ">= 3.0", "< 5.3"
 
   spec.add_development_dependency "rails", ">= 4.2", "< 5.3"
-  spec.add_development_dependency "bundler", "~> 1.6"
+  spec.add_development_dependency "bundler"
```

Comment:
Can you make that `~> 1` or `>= 1.6` instead? This change would allow Bundler to downgrade bundler to 0.x in some cases, and I do want to know that it's 1.x/1.6 or later.